factory-configurator-client/psa-driver/source/psa_driver_crypto.c

```diff
@@ -14,6 +14,7 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------
 #ifdef MBED_CONF_MBED_CLOUD_CLIENT_PSA_SUPPORT
+#include "platform/mbed_version.h"
```

Comment:
@ccli8 
I don't see any of the #define in this file being used: https://github.com/ARMmbed/mbed-os/blob/master/platform/include/platform/mbed_version.h 
What compile errors are you seeing without this include file?

Reply:
@marcuschangarm I met link error with undefined `mbedtls_psa_crypto_free` symbol:

```
Compile [100.0%]: psa_driver_crypto.c
[Warning] psa_driver_crypto.c@547,5: implicit declaration of function 'mbedtls_psa_crypto_free' is invalid in C99 [-Wimplicit-function-declaration]
Link: mbed-cloud-client-example
[Warning] @0,0: L3912W: Option 'legacyalign' is deprecated.
[Error] @0,0: L6218E: Undefined symbol mbedtls_psa_crypto_free (referred from BUILD/NU_M2354/ARM/mbed-cloud-client/factory-configurator-client/psa-driver/source/psa_driver_crypto.o).
```

Checking the lines below, `MBED_MAJOR_VERSION` is not available, which is defined in `mbed_version.h`:
https://github.com/PelionIoT/mbed-cloud-client/blob/d7edc529ed3722c811ff401440ef58ea980bf543/factory-configurator-client/psa-driver/source/psa_driver_crypto.c#L543-L548